lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+created: 07/30/2020
+---
+# Summary
+This LIP describes the conceptual model and implementation of validator configuration management on the Libra blockchain.
+
+# Abstract / Motivation
+The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins that supports a regulated global digital payment network. Both the blockchain and the stablecoin financial instruments are controlled by Libra Networks, Ltd. (“Libra Networks” or “LN”) a subsidiary of the Libra Association  (the “Association” or “LA”).  LPN operates under a payment system license from FINMA (the “FINMA License”), and LN is responsible for ensuring that LPN operates in compliance with the FINMA License.
+
+The Libra Association is an independent membership organization that is responsible for the governance of the LPN and the development of the Libra project. The Association is governed by the Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board. 
+
+Each member that participates in maintaining the blockchain by operating a Validator Node or designating a party to operate a Validator Node is referred to as a Validator Owner. The Validator Set defines the current set of Validator Nodes that run consensus protocol. Validator Nodes that are currently part of the Validator Set process transactions and interact with each other to reach consensus on the state of the blockchain. The Validator Set may change from time to time and not all members may have Validator Nodes in the Validator Set at any given time. Reasons for changing the Validator Set include joining or departure of members from the Association and actions needed for compliance reasons or to comply with LN’s obligations under its payment system license from FINMA to guarantee the secure and performant operation of the platform.
+
+# Terminology
+* *Validator Node* ("node") is a replica that runs the Libra Protocol to maintain a database of programmable resources. Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions. 
+* *Validator Owner* is a entity that manages an on-chain Libra account of the member of the Libra Association. The account stores the *Validator Config* resource that contains information about the Validator Node of this Validator Owner. This information includes *consensus public key* (EdDSA public key that is used by others to authenticate consensus messages of this node) and *networking information* (network addresses and static Noise keys for establishing secure connections with this node from other nodes or from clients).
```

Comment:
"Validator Owner" is an off-chain concept that is beyond the scope of the Move code.  Enforcing the 1:1 relationship between validator owners and validator nodes is a  rule that is enforced administratively off-chain.

Reply:
Not quite sure I understand your point.
I define the Validator Owner as an on-chain Libra Account, while the Validator Node is a replica (is a server that runs a consensus node). I think the 1:1 relation between the Validator Owner and the Validator Node is enforced by the fact that a single Validator Owner stores a single Validator Config resource that essentially defines the Validator Node (it contains consensus pk, network addresses).

Reply:
I agree with Lera: We want Validator Owners representation on chain. We used to (and I believe, still currently) call these accounts simply Validators;  I recommend to use Validator Owner. Albeit slightly longer, it adheres to the language used in the Association legal frameworks, and also disambiguates